### PR TITLE
[SD-1146] Fix compare at price displaying on PDP

### PR DESCRIPTION
**Pull request by aplegatt**

### CODE REVIEW

File: core/app/helpers/spree/products_helper.rb [hunk 1]
```diff
@@ -35,6 +35,11 @@ def default_variant(variants, product)
       variants_option_types_presenter(variants, product).default_variant || product.default_variant
     end
 
+    def should_display_compare_at_price(default_variant)
+      default_variant_price = default_variant.prices.find_by(currency: current_currency)
```
Comment (damianlegawiec) [inline core/app/helpers/spree/products_helper.rb:39]:
```suggestion
      default_variant_price = default_variant.price_in(current_currency)
```

Comment (damianlegawiec) [reply core/app/helpers/spree/products_helper.rb:39]: https://guides.spreecommerce.org/developer/core/products.html#multi-currency-support

File: core/app/helpers/spree/products_helper.rb [hunk 2]
```diff
@@ -35,6 +35,11 @@ def default_variant(variants, product)
       variants_option_types_presenter(variants, product).default_variant || product.default_variant
     end
 
+    def should_display_compare_at_price(default_variant)
```
Comment (przemosk) [inline core/app/helpers/spree/products_helper.rb:38]: if this method return `true/false` we could change name to: `should_display_compare_at_price?(default_variant)` wdyt ?

**Review by przemosk — APPROVED** [review]
☀️

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**